packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -158,6 +158,38 @@ function getCommonEventData<T extends keyof DocumentEventMap>(eventConfig: UIEve
   };
 }
 
+
+// Keep track of event handlers
+const uiEventHandlers = new Map<UIEventConfig['eventName'], {
+  captureHandler: (event: Event) => void,
+  bubbleHandler: (event: Event) => void,
+  active: boolean,
+}>;
+
+export function disableEventHandlers(eventName: UIEventConfig['eventName']): void {
+  const handlerConfig = uiEventHandlers.get(eventName);
+  if (handlerConfig?.active === true) {
+    window.document.removeEventListener(eventName, handlerConfig.captureHandler, true);
+    window.document.removeEventListener(eventName, handlerConfig.bubbleHandler, false);
+    uiEventHandlers.set(eventName, {
```

Comment:
although this is correct, but create extra objects, you could also just set active to false. Unless, you define the data in the map as readonly to be more safe.